### PR TITLE
i1166 - simplify vault ssl bootstrap process

### PR DESCRIPTION
**Pull request by richfitz**
The core of the idea is in ssl-key/README.md.  This is as far as I can see 

I think this is a bit simpler, and it will eliminate the need to unseal the vault twice (and noone needs to do it over loopback).  I've removed all the bootstrap bits that I can find.

If this looks good I can make the corresponding changes in https://github.com/vimc/montagu-recovery/pull/1 / VIMC-423

You can test everything locally except actually connecting to the vault that this creates (because you will have the wrong hostname)

### CODE REVIEW

**Review by MartinEden — COMMENTED** [review]
This seems generally okay, but I'm feeling a bit fried at the moment, so I haven't given it a really careful line-by-line review.

It would be good to add some documentation (maybe to the new starter page on the wiki, maybe in the main Montagu repo) to remind people to add new keys to this repo.

**Comment by richfitz** [comment]
The docs here should be a bunch clearer now

**Comment by MartinEden** [comment]
I've tested this and it works great - much simpler than before to get the Vault running. Just one small change needed which I will list inline.

File: scripts/decrypt-ssl-key.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+FILE_ENC=/vault/config/ssl_private_key.enc
+FILE_CLEAR=/vault/config/ssl_private_key
+echo "On your PC, run"
+echo
+echo "  git clone https://github.com/vimc/montagu-vault"
+echo "  ./montagu-vault/ssl-key/decrypt-key.sh <your-key-name>"
+echo
+echo "where <your-key-name> corresponds to an entry in ssl-keys/pubkeys/"
```
Comment (MartinEden) [inline scripts/decrypt-ssl-key.sh]: Path is wrong, should be `ssl-key/pubkey/` not `ssl-keys/pubkeys`

**Review by MartinEden — CHANGES_REQUESTED** [review]

File: scripts/decrypt-ssl-key.sh (hunk 1 above)
Comment (richfitz) [reply scripts/decrypt-ssl-key.sh]: Thanks - updated now

**Comment by richfitz** [comment]
I think this is probably good to go now, then onto https://github.com/vimc/montagu-recovery/pull/1 followed by https://github.com/vimc/montagu/pull/58

**Comment by richfitz** [comment]
After merge, I will edit https://github.com/vimc/vimc-wiki/wiki/New-users%27-set-up

**Review by MartinEden — APPROVED** [review]

**Pull request merged**